server/game/cards/09.2-AGE/LightingTheFuse.js

```diff
@@ -0,0 +1,68 @@
+const ActionCard = require('../../actioncard.js');
+const PhaseNames = require('../../Constants/PhaseNames.js');
+const GameActions = require('../../GameActions/index.js');
+
+class LightingTheFuse extends ActionCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Lighting the Fuse',
+            playType: 'cheatin resolution',
+            target: {
+                activePromptTitle: 'Select deed to blow up',
+                cardCondition: {
+                    location: 'play area',
+                    condition: card => this.controller.getOpponent().equals(card.owner)
+                },
+                cardType: 'deed'
+            },
+            handler: context => {
+                const thecheater = context.player.getOpponent();
+                const kaboomification = () =>
+                    this.game.resolveGameAction(GameActions.ReturnCardToHand({ card: context.target }), context).thenExecute(() => {

```

Comment:
Game actions start with lower case.
```suggestion
                    this.game.resolveGameAction(GameActions.returnCardToHand({ card: context.target }), context).thenExecute(() => {

```